jinahub/encoders/audio/VGGISHAudioEncoder/requirements.txt

```diff
@@ -1,7 +1,7 @@
 jina-commons @ git+https://github.com/jina-ai/jina-commons.git#egg=jina-commons
 jina[hub]==2.0.7
```

Comment:
remove `jina` depedency from requirements.txt

Reply:
Addressed. Please see [here](https://github.com/jina-ai/executors/pull/31/commits/adc062852447ab0fa23e4b5dbc3cbcdba9e083ff) for the change, thank you.